### DISCUSSION TITLE
tailwind ecommerce store navigation code is note working it is magnifying whenver it is launched with react app

### DISCUSSION THREAD

**Discussion by adarsh180** [tailwindlabs/tailwindcss#12918] [Help]
https://github.com/adarsh180/Website.git

check the git and please answer my query 
![Screenshot (12)](https://github.com/tailwindlabs/tailwindcss/assets/87196890/01d651c3-b499-4384-ae81-0728124e7fa0)
![Screenshot (11)](https://github.com/tailwindlabs/tailwindcss/assets/87196890/dd340bd0-5e32-43dc-8aa3-3ae4e9b62cdf)

**Comment by wongjn** [comment]
The file that has Tailwind class names, `src/customer/components/Navigation/Navigation.jsx`, is not covered by the `content` file globs in the Tailwind configuration. Consider altering your Tailwind configuration like:

```diff
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{html,js}"],
+  content: [
+    "./src/**/*.{html,js}",
+    "./src/customer/components/Navigation/Navigation.jsx",
+  ],
   theme: {
     extend: {},
```
Or perhaps:
```diff
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{html,js}"],
+  content: ["./src/**/*.{html,js,jsx}"],
   theme: {
     extend: {},
```
[Read more about content configuration in Tailwind](https://tailwindcss.com/docs/content-configuration).
